crates/graphql/src/schema/objects/listing_receipt.rs

```diff
@@ -26,21 +30,24 @@ impl<'a> TryFrom<models::ListingReceipt<'a>> for ListingReceipt {
             trade_state_bump,
             created_at,
             canceled_at,
-            bookkeeper: _,
-            purchase_receipt: _,
-            token_size: _,
-            bump: _,
-            ..
+            bookkeeper,
+            purchase_receipt,
+            token_size,
+            bump,
         }: models::ListingReceipt,
     ) -> Result<Self, Self::Error> {
         Ok(Self {
+            token_size: token_size.try_into().unwrap(),
             address: address.into_owned(),
             trade_state: trade_state.into_owned(),
             seller: seller.into_owned(),
             metadata: metadata.into_owned(),
             auction_house: auction_house.into_owned(),
             price: price.try_into()?,
             trade_state_bump: trade_state_bump.into(),
+            bookkeeper: bookkeeper.into_owned(),
+            purchase_receipt: purchase_receipt.map(|pr| pr.into_owned()),
+            bump: bump.into(),
```

Comment:
should be below created_at and canceled_at

Reply:
what should be? all 3 fields? why?